invenio_users_resources/administration/views/users.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio administration users view module."""
+from flask import current_app
+from invenio_administration.views.base import (
+    AdminResourceDetailView,
+    AdminResourceListView,
+)
+from invenio_i18n import lazy_gettext as _
+
+
+class UsersListView(AdminResourceListView):
+    """Configuration for users sets list view."""
+
+    api_endpoint = "/users"
+    name = "users"
+    resource_config = "users_resource"
+    title = "User management"
+    menu_label = "Users"
+    category = "User management"
+    pid_path = "id"
+    icon = "exchange"
+
+    display_search = True
+    display_delete = False
+    display_edit = False
+    display_create = False
+
+    item_field_list = {
+        "username": {"text": _("Username"), "order": 1},
+        "email": {"text": _("Email"), "order": 2},
+        "created": {"text": _("Created"), "order": 4},
+        "updated": {"text": _("Updated"), "order": 5},
+    }
```

Comment:
Should we add `active` and `confirmed_at` to the field list?
@ppanero mentioned they are very useful to have in Zenodo administration.

Reply:
At the moment, adding them to this list would not bring anything new, since the list of users retrieved are only the `created=True and confirmed=True`. 
If we want to add those two fields to the table, we probably should change also the https://github.com/inveniosoftware/invenio-users-resources/blob/master/invenio_users_resources/services/users/service.py#L70 to remove that filter.

Reply:
I'll follow @kpsherva  suggestion, and add those fields (even if at the moment they will always be true). The behavior can be changed later on